dotCMS/src/main/java/com/dotcms/rest/api/v1/secret/ServiceIntegrationHelper.java

```diff
@@ -324,23 +325,40 @@ private void validateIncomingParams(final Map<String, Param> incomingParams, fin
         for (final Entry<String, Param> incomingParamEntry : incomingParams.entrySet()) {
             final String incomingParamName = incomingParamEntry.getKey();
             final Param describedParam = serviceDescriptorParams.get(incomingParamName);
+            final Param incomingParam = incomingParamEntry.getValue();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4979325)